src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt

```diff
@@ -66,7 +64,7 @@ object ScoreCalculation {
         "F4" to FloorRequirement(.6, 12 * 60),
         "F5" to FloorRequirement(.7),
         "F6" to FloorRequirement(.85, 12 * 60),
-        "F7" to FloorRequirement(speed = 14 * 60),
```

Comment:
Supposedly they changed the requirement to 14 minutes

Reply:
I played couple of f7 runs but still 12 mins

Reply:
bruh what

Reply:
I think they said on 1/6 they rolled it out

Reply:
just checked, it's 12